dockers/docker-dhcp-relay/Dockerfile.j2

```diff
@@ -0,0 +1,31 @@
+FROM docker-config-engine
+
+# Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Update apt's cache of available packages
+RUN apt-get update
+
+{% if docker_dhcp_relay_debs.strip() -%}
+# Copy built Debian packages
+{%- for deb in docker_dhcp_relay_debs.split(' ') %}
+COPY debs/{{ deb }}{{' '}} debs/
+{%- endfor %}
+
+# Install built Debian packages and implicitly install their dependencies
+{%- for deb in docker_dhcp_relay_debs.split(' ') %}
+#RUN dpkg -i debs/{{ deb }}
```

Comment:
it looks like commented code. it's confusing. Someone didn't removed old code, or necessary code wasn't uncommented.